docs/server-side-configuration.md

```diff
@@ -0,0 +1,37 @@
+
+---
+
+# Cozy MyAccounts: Server side configuration
+
+## Authorized categories list
+
+The MyAccounts categories names list won't be automatically computed from available connectors in order to keep it clear and relevant for all Cozy users with a growing number of connectors.
+
+Instead, they need to be configured on the served side, in a simple way, from a config file. This is a list of categories authorized to be used:
+
+```javascript
+{
+    authorizedCategories:
+        {
+            'insurance': 1,
```

Comment:
I'd start with a just an array of slugs. It is what is easier to read & edit.

> Lookup in a list is more readable using an object than an array:

Not really. ES7 has `includes`. Coffescript gives you:

```coffee
if not category in authorizedCategories
    # do something
```

And the right way to do a lookup on an object is:

```coffee
if authorizedCategories.hasOwnProperty(category)
    # do something
```

Else, wait for the day someone will create a konnector with the [`watch`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/watch) category :see_no_evil:

> For performance reason.

Well, if a konnector can be in hundeds or thousands of categories, an object will be faster than an array. But, with very few categories, it's not the case.